<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="rahulshettyacademy.tests.ErrorValidationsTest">
        <methods>
          <include name="LoginErrorValidation"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- rahulshettyacademy.tests.ErrorValidationsTest -->
      <class name="rahulshettyacademy.tests.SubmitOrderTest">
        <methods>
          <include name="tearDown"/>
          <include name="OrderHistoryTest"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- rahulshettyacademy.tests.SubmitOrderTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
